### PR TITLE
Add New Issue template.

### LINKED ISSUE #111: Should we have a "ISSUE_TEMPLATE.md" for Glue?
I noticed that Glue does not have the same `ISSUE_TEMPLATE.md` as other hapi repos. I think it probably should, so that we can get annoyed when people clear it out ;) 

But in all seriousness, I think most of the time it does help.

### PR DESCRIPTION
**Pull request by WesTyler**
Just in case the answer is "yes", closes #111 .

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+#### What are you trying to achieve or the steps to reproduce ?
+
+Describe your issue here, include manifests and plugin objects if needed.
+
+```js
+const manifest = {};
+```
+
+#### Which result you had ?
```
Comment (csrl) [inline .github/ISSUE_TEMPLATE.md]: "What result did you get ?"

Comment (csrl) [reply .github/ISSUE_TEMPLATE.md]: Seems the other hapi repos share that odd phrasing...

**Comment by csrl** [comment]
I guess we should also bring in CONTRIBUTING doc as well to stay consistent.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (WesTyler) [reply .github/ISSUE_TEMPLATE.md]: Yeah, I debated with myself on fixing the phrasing or staying consistent. I don't have strong opinions in either direction.

**Comment by WesTyler** [comment]
Sure thing

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (csrl) [reply .github/ISSUE_TEMPLATE.md]: I'd fix it - the main hapi repo has corrected it.

Comment (WesTyler) [reply .github/ISSUE_TEMPLATE.md]: Done

**Comment by WesTyler** [comment]
I believe so. @csrl there wasn't anything else you were waiting on from me, was there?

**Comment by csrl** [comment]
No, not sure why this slipped through the cracks.  Thanks for the ping.

**Pull request merged**